src/OpenLoco/Windows/ToolbarTopAlt.cpp

```diff
@@ -163,18 +164,19 @@ namespace OpenLoco::Ui::Windows::ToolbarTop::Editor
                 break;
 
             case 5:
-            {
-                loco_global<uint8_t, 0x00508F16> _screenshotCountdown;
-                _screenshotCountdown = 10;
+                Input::triggerScreenshotCountdown(10, 0);
+                break;
+
+            case 6:
```

Comment:
Its high time we defined some constants for these.

Reply:
Perhaps in the same way as the main ToolbarTop window handles these. Good Hacktoberfest issue?